1_fetch/src/get_nwis_data.R

```diff
@@ -41,17 +42,16 @@ download_nwis_site_data <- function(site_num, out_file, parameterCd = '00010', s
   }
   # -- end of do-not-edit block
   
-  write_csv(data_out, file = out_file)
-  return(out_file)
+  return(data_out)
 }
 
-concat_data <- function(files_in){
+concat_data <- function(data_in, out_file){
   data_out <- data.frame()
   # loop through files to download 
-  for (download_file in files_in){
+  for (these_data in data_in){
```

Comment:
No need to make this change, just wanted to note that now you're passing in a list of dataframes instead of a list of filepaths, you can get away without a for loop, by simply calling:
```r
concat_data <- function(data_in, out_file) {
    bind_rows(data_in) %>%
        write_csv(out_file)

    return(out_file)
}
```